web/src/app/projects/repository/repository.component.scss

```diff
@@ -6,3 +6,77 @@ mat-list {
   overflow: hidden;
   text-overflow: ellipsis;
 }
+
+.dashboard-card.mat-card-content{
+  margin:0 !important;
```

Comment:
Can we try not to use `!important`

Reply:
Removed from here as it is not required now. But at other places I need !important as Its overriding the default scss.

Reply:
I think it would be better to put an `id` on the component and target from there, I think this will take higher priority